pkg/miner/micro_miner.go

```diff
@@ -64,74 +65,119 @@ func (a *MicroMiner) Micro(minedBlock *proto.Block, rest proto.MiningLimits, key
 		parentTimestamp = parent.Timestamp
 	}
 
-	//
-	txCount := 0
+	txCount := 0 // counter for successfully applied transactions
 	binSize := 0
+	droppedTxCount := 0
 
 	var appliedTransactions []*types.TransactionWithBytes
 	var inapplicable []*types.TransactionWithBytes
 	var txSnapshots [][]proto.AtomicSnapshot
 
-	_ = a.state.Map(func(s state.NonThreadSafeState) error {
+	_ = a.state.MapUnsafe(func(s state.NonThreadSafeState) error {
```

Comment:
@nickeskov, I thought you wanted to remove `MapUnsafe`. Did you decide to keep it?